azure-pipelines.yml

```diff
@@ -83,6 +83,23 @@ stages:
     steps:
       - template: .azure-pipelines/impacted_area_testing/get-impacted-area.yml
 
+  - job: choose_between_mixed_and_py3_ptf_image
+    displayName: "Choose between latest (mixed type) and py3 ptf image"
+    timeoutInMinutes: 10
+    continueOnError: false
+    pool: sonic-ubuntu-1c
+    steps:
+      - script: |
+          import random
+          tag = 'latest'
+          if random.random() < 0.3:
+            tag = 'py3only'
```

Comment:
This pipeline yaml in master branch is also used by PR testing for the release branches.
For old release branches, they do not support py3only. So setting ptf_imagetag should comply with the branch.

Reply:
Thanks for the comment. 

 - `sonic-mgmt` PR checkers for the release branches use the `azure-pipelines.yml` on the respective branch and only reference the `run-test-elastictest-template.yml` on the master branch. For instance [202305](https://github.com/sonic-net/sonic-mgmt/blob/202305/azure-pipelines.yml#L68)

Since this file is only run for PR checkers when a merge happens to `master`. I think this change shouldn't impact other branches.